code/modules/mod/modules/modules_general.dm

```diff
@@ -655,3 +655,108 @@
 		has_tk = FALSE
 		return
 	mod.wearer.dna.remove_mutation(TK)
+
+//Special module for the Captain's Magnate suit, and the CC Corporate suit.
+//Literally only a thing so that https://pipe.miroware.io/5b52ba1d94357d5d623f74aa/mspfa/Nuke%20Ops/Panels/0648.gif can be real
+/obj/item/mod/module/hat_stabilizer
+	name = "MOD hat stabilizer module"
+	desc = "A simple set of deployable stands, directly atop one's head; \
+		these will deploy under a select few hats to keep them from falling off, allowing them to be worn atop the sealed helmet. Unfortunately, you still need to take off the hat first to deploy the helmet itself. \
+		This is a must-have for Nanotrasen Captains, enabling them to show off their authoritative hat even while in their MODsuit."
+	icon_state = "hat_holder"
+	module_type = MODULE_PASSIVE //The description says it auto-deploys for certain hats; that's just an excuse to have a whitelist of items.
+	complexity = 0 //Its FREE!
+	active_power_cost = 0 //Yes, thats right, FREE!
+	removable = FALSE
```

Comment:
Why unremovable

Reply:
0 complexity modules usually arent removable

Reply:
as above

up for debate though: I may make it a maintenance module as well, or at the least removable so you can steal the captain's hat(MOD)

Reply:
Actually made it removable and appear in maint loot, if there's an objection I can revert that easy though